src/ViewModel/ContentHeader.php

```diff
@@ -26,13 +26,16 @@
     private $licence;
     private $audioPlayer;
 
-    const XX_SHORT = 'xx-short';
-    const X_SHORT = 'x-short';
-    const SHORT = 'short';
-    const MEDIUM = 'medium';
-    const LONG = 'long';
-    const X_LONG = 'x-long';
-    const XX_LONG = 'xx-long';
+    const LENGTH_LIMITS = [
+        19 => 'xx-short',
+        38 => 'x-short',
+        46 => 'short',
+        57 => 'medium',
+        118 => 'long',
+        155 => 'x-long',
+    ];
+
+    const MAX_LENGTH = 'xx-long';
```

Comment:
Separated `MAX_LENGTH` from the `LENGTH_LIMITS` array as it's the only one not designated by a maximum character count.